containersQa/testlib.bash

```diff
@@ -924,4 +928,37 @@ function newUserCheck() {
 EOF
     $PD_PROVIDER build -f $podmanfile .
 
-}
\ No newline at end of file
+}
+
+function setAlgorithmTestsUrlVars {
+  checkAlgorithmsUrl="https://raw.githubusercontent.com/rh-openjdk/jtreg-buffer/main/test/reproducers/checkAlgorithms/CheckAlgorithms.java"
+  cipherListUrl="https://raw.githubusercontent.com/rh-openjdk/jtreg-buffer/main/test/reproducers/1906862/CipherList.java"
+}
```

Comment:
One of the things we are trying to do with the container testing is remove the in-test pulls from the internet. I know there are existing examples where this is the case, that will be handled eventually. But any new content should be added under source control so that it can be tracked.  As this is right now, there is no way to track which version of these files would be used for execution. The end result is I could run this test two times in a row and technically get different results. Please remember these tests are run at a much higher frequency than traditional RPM tests.

Reply:
> This cannot be added like this. We cannot rely on sources from other repositories at their head. This has broken testing in the past.

Wdyt, about having check on download, and if it fails, the test would be marked as skipped?

Reply:
> Better yet, these files should be added to the source code with the idea everytime they are updated in jtreg-buffer, a new PR is opened here.

I'm not going to oppose you. If you insists, it will happen. Just my note would e that everything is better then duplicated code. And the .java file is not exactly simple, and will be adapting to each jdk version and each rhel crypto policies adjust .

Reply:
> One of the things we are trying to do with the container testing is remove the in-test pulls from the internet. I know there are existing examples where this is the case, that will be handled eventually. But any new content should be added under source control so that it can be tracked. As this is right now, there is no way to track which version of these files would be used for execution. The end result is I could run this test two times in a row and technically get different results. Please remember these tests are run at a much higher frequency than traditional RPM tests.

I'm ok to change it back to exact revision as Patrik had it.  Patrik, if you will do so, please, set the revision id to variable (it was duplicated in original commit)

Reply:
No no. @judovana You are missing my point. I do not want the testcode downloading code it does not need to. If the testcode is in the repo then the test execution becomes more stable. While I agree with you in-principle about the duplicate code, the only other options is to break that code out into a third repo and make it a library. I do not want this testsuite to be pulling testcode from other repos during the test execution.